atomic_reactor/plugins/post_tag_by_labels.py

```diff
@@ -66,6 +66,11 @@ def get_label(label_name):
         version = get_label("version")
         release = get_label("release")
 
+        # docker tags cannot be longer than 128 chars
+        # see https://github.com/docker/docker/pull/8447
+        if len(release) > 128:
+            release = release[:127]
```

Comment:
Fixed. Do we want to print the resulting tag? It would be visible after pulp push anyway, but it might make sense to print it early in the log